stix_shifter_modules/elastic_ecs/stix_transmission/api_client.py

```diff
@@ -95,10 +95,8 @@ async def search_pagination(self, query_expression, lastsortvalue=None, length=D
         return await self.client.call_api(endpoint, 'GET', headers, data=json.dumps(data), timeout=self.timeout)
 
     async def get_max_result_window(self):
-        # GET winlogbeat-*/_settings?include_defaults=true
-        endpoint = self.setting_endpoint
-        endpoint = "{}?include_defaults=true".format(endpoint)
-        return await self.client.call_api(endpoint, 'GET', timeout=self.timeout)
+        max_result_window_url = self.setting_endpoint + "/index.max_result_window?include_defaults=true"
+        return self.client.call_api(max_result_window_url, 'GET', timeout=self.timeout)
```

Comment:
Please add await before calling self.client.call_api
Otherwise you will get coroutine was never awaited error

Reply:
@baulus Done in 2e6a95edc79bf420f82b1b795ca42812af943636